frontend/src/index.tsx

```diff
@@ -58,14 +58,6 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                   </ProtectedRoute>
                 ),
               },
-              {
-                path: AppRoute.SIGN_IN,
-                element: <Auth />,
-              },
-              {
-                path: AppRoute.SIGN_UP,
-                element: <Auth />,
-              },
```

Comment:
Maybe I'm missing smth, but why are these routes just deleted?

Reply:
I have deleted the duplicates